packages/common/src/store/notifications/types.ts

```diff
@@ -99,20 +99,78 @@ export type FollowNotification = BaseNotification & {
   userIds: ID[]
 }
 
+export type FollowPushNotification = {
```

Comment:
can we define a "PushNotification" type that covers most of the stuff like, initiator, blocknumber, timestamp etc, and also have an Action type, and then we fill in the specifics for each one?

Reply:
maybe it's fine as is to be as explicit as possible? what you think?

Reply:
I wanted to, but there isn't enough consistency in the structures and I can't see them all so will leave as is for now. Good thought though, may come back to this